contents/handbook/engineering/developing-locally.md

```diff
@@ -78,6 +80,8 @@ In case some steps here have fallen out of date, please tell us about it – fee
 
     ClickHouse and Kafka won't be able to talk to each other without these mapped hosts.
 
+    > If running Podman, `/etc/hosts` could be used as the base hosts file for containers by default. This can make hostname resolution fail in the ClickHouse container, and can be mended by setting `base_hosts_file="none"` in [`containers.conf`](https://github.com/containers/common/blob/main/docs/containers.conf.5.md#containers-table). This can be an issue only in newer versions of the OCI runtime, as the change was introduced in April 2022.
```

Comment:
Maybe "If you're using Podman instead of Docker"`, as not _everyone_ might know that Podman is a Docker replacement

Reply:
And makes more sense to add the extra clarification once you have dropped the initial paragraph :+1: